### PR TITLE
[AIRFLOW-XXX] Name default config_file param in KubernetesPodOperator docstring

### PR DESCRIPTION
**Pull request by leahecole**
Wasn't able to re-open #5138 after the rebase

### CODE REVIEW

File: airflow/contrib/operators/kubernetes_pod_operator.py [hunk 1]
```diff
@@ -71,7 +71,8 @@ class KubernetesPodOperator(BaseOperator):
     :type affinity: dict
     :param node_selectors: A dict containing a group of scheduling rules
     :type node_selectors: dict
-    :param config_file: The path to the Kubernetes config file
+    :param config_file: The path to the Kubernetes config file.
+        If not specified, defaults to ``~/.kube/config``
```
Comment (zhongjiajie) [inline airflow/contrib/operators/kubernetes_pod_operator.py]:
```suggestion
        If not specified, using defaults value ``~/.kube/config``
```

**Review by zhongjiajie — COMMENTED** [review]
This may be better.

File: airflow/contrib/operators/kubernetes_pod_operator.py (hunk 1 above)
Comment (leahecole) [reply airflow/contrib/operators/kubernetes_pod_operator.py]: I adjusted to a modified version of your suggestion that has more correct grammar - PTAL once travis runs :)

Comment (zhongjiajie) [reply airflow/contrib/operators/kubernetes_pod_operator.py]: Looking great.

Comment (leahecole) [reply airflow/contrib/operators/kubernetes_pod_operator.py]: Great. Let me know if anything else needs to be done to make it merge-able! :)

Comment (zhongjiajie) [reply airflow/contrib/operators/kubernetes_pod_operator.py]: You should remind committer to review your change.

Comment (zhongjiajie) [reply airflow/contrib/operators/kubernetes_pod_operator.py]: @XD-DENG @potiuk PTAL if you have time. thx

**Comment by leahecole** [comment]
friendly ping to @potiuk  - had to reopen this in a new PR - had previously been approved by you

**Comment by dimberman** [comment]
LGTM

**Pull request merged**